{{cookiecutter.project_slug}}/.github/workflows/ci.yaml

```diff
@@ -0,0 +1,98 @@
+---
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+
+  test:
+    name: Unit tests
+    strategy:
+      matrix:
+        python-version: [2.7, 3.6, 3.7]
+        os: [macos-latest, ubuntu-latest, windows-latest]
+
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: install python dependencies
+        run: |
+          python -m pip install --upgrade pip
+{%- if cookiecutter.package_requirements != "Type any packages required, leaving spaces in between each name. Press enter if none." %}
```

Comment:
Is there not a way to default this to `None` rather than checking for an exact string match?

Reply:
Fair point. I honestly don't know. I took it from the existing template.